practices/team_culture.md

```diff
@@ -0,0 +1,87 @@
+---
+slug: team-culture
+stages:
+  - building_team
+short_description: A team culture is how people work together towards a common goal with team members sharing their values, beliefs, attitudes and behaviours.
+tags:
+  - team culture
+  - team
+  - team members
+  - common goal
+  - values
+  - attitudes
+  - team work
+keywords:
+  - team culture
+  - team
+  - team members
+  - common goal
+  - values
+  - beliefs
+---
+
+# Team Culture
+
+**TL;DR**
+
+Team Culture is an essential part of any organization and it emphasizes on achieving a common goal by sharing values, beliefs, attitudes and behaviours among team members.
+
+## What Is Team Culture?
+
+Team Culture is not dependant on or created by an individual member's personality, but rather built on how the team collectively functions.
+
+The definition of team culture is fairly simple as it implies each member of a team should work towards achieving a common goal with collective efforts, discussions and feedback.
+
```

Comment:
I know that you mention it in the TL;DR, but it could be interesting if you could expand on it.

Reply:
> I know that you mention it in the TL;DR, but it could be interesting if you could expand on it.

Actually I didn't know what TL;DR meant and I just checked what it means. Should I remove TL;DR? or do you want me to  just write something more under Team Culture Heading.

Reply:
Hello, please, keep the TL;DR. It is a short explanation of the whole article (so, you can see if this article covers practice/problem you are interested in). 

I suggested (and sorry for not being clear) to add into Team culture more about values and beliefs, goals or behaviors.

Reply:
> Hello, please, keep the TL;DR. It is a short explanation of the whole article (so, you can see if this article covers practice/problem you are interested in).
> 
> I suggested (and sorry for not being clear) to add into Team culture more about values and beliefs, goals or behaviors.

Ok then I will do as you suggested.